tests/migrations/test-migration-Mig004.py

```diff
@@ -0,0 +1,41 @@
+import os
```

Comment:
Please change the module name to comply with PEP8 (underscores instead of dashes)
https://peps.python.org/pep-0008/#package-and-module-names

Reply:
ok